src/test/java/seedu/address/ui/CommandIntegrationTest.java

```diff
@@ -0,0 +1,108 @@
+//package seedu.address.ui;
```

Comment:
I believe this file should be deleted (its all commented out)? Could you confirm.